### ISSUE TITLE
Is it possible to define a template and custom variables

### ISSUE THREAD

**Issue by avivash** [dongri/mailgun-rs#4]
Thanks for putting this together. The JS mailgun API allows you to specify a `template`  and custom variables that can be read within the mailgun template:

```ts
        to: email,
        subject: `Welcome to ${appName}`,
        template: 'recovery',
        'v:email': email,
```

Is it possible to do the same with this library? I don't see them in the `Message` struct:

```rs
pub struct Message {
    pub to: Vec<EmailAddress>,
    pub cc: Vec<EmailAddress>,
    pub bcc: Vec<EmailAddress>,
    pub subject: String,
    pub text: String,
    pub html: String,
}
```

**Comment by dongri** [comment]
@avivash Thank you
https://github.com/dongri/mailgun-rs/pull/5

published v0.1.5
https://crates.io/crates/mailgun-rs

**Comment by avivash** [comment]
> @avivash Thank you #5
> 
> published v0.1.5 https://crates.io/crates/mailgun-rs

Amazing! Thanks for getting that updated so quickly 🙏🏼
